tests/scripts/minikube.sh

```diff
@@ -16,6 +16,85 @@ EOF
     minikube start --memory=3000 --kubernetes-version ${KUBE_VERSION} --extra-config=apiserver.Authorization.Mode=RBAC
 }
 
+# workaround for kube-dns CrashLoopBackOff issue with RBAC enabled
+#issue https://github.com/kubernetes/minikube/issues/1734 and https://github.com/kubernetes/minikube/issues/1722
+enable_roles_for_RBAC() {
+    cat <<EOF | kubectl create -f -
+# Wide open access to the cluster (mostly for kubelet)
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: cluster-writer
+rules:
+  - apiGroups: ["*"]
+    resources: ["*"]
+    verbs: ["*"]
+  - nonResourceURLs: ["*"]
+    verbs: ["*"]
+---
+# Full read access to the api and resources
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: cluster-reader
+rules:
+  - apiGroups: ["*"]
+    resources: ["*"]
+    verbs: ["get", "list", "watch"]
+  - nonResourceURLs: ["*"]
+    verbs: ["*"]
+---
+# Give admin, kubelet, kube-system, kube-proxy god access
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: cluster-write
+subjects:
+  - kind: User
+    name: admin
+  - kind: User
+    name: kubelet
+  - kind: ServiceAccount
+    name: default
+    namespace: kube-system
+  - kind: User
+    name: kube-proxy
+roleRef:
+  kind: ClusterRole
+  name: cluster-writer
+  apiGroup: rbac.authorization.k8s.io
+---
+# Setup sd-build as a reader. This has to be a
```

Comment:
why do we need these `sd-*` accounts, roles and bindings?  in the original issue https://github.com/kubernetes/minikube/issues/1734, it looks like the sample fix came from the Screwdriver project: http://blog.screwdriver.cd/post/161863341372/set-up-screwdriver-in-kubernetes, which is where I think this `sd-*` stuff is coming from.

We aren't using screwdriver, so we shouldn't add those RBAC configs to our scripts, right?  I would *think* that only cluster-writer and cluster-reader are necessary.  Can you please try this scenario after removing all the `sd-*` stuff that I suspect isn't necessary?

Reply:
thanks for catching that,i just coped the roles from the last comment in the issue. Cleaned them up now.

Reply:
@dangula can you confirm that kube-dns worked OK with RBAC enabled once you removed those `sd-*` permissions?

Reply:
@jbw976 it worked, i ran full regession and all tests passed.

Reply:
ah, it looks like the cluster-reader role isn't used anymore now.  it was only used by the sd-build account.  let's get rid of the cluster-reader role too then I think this change will be complete.  thanks @dangula